katharsis-servlet/src/test/java/io/katharsis/servlet/resource/model/Locale.java

```diff
@@ -0,0 +1,36 @@
+package io.katharsis.servlet.resource.model;
+
+import io.katharsis.resource.annotations.JsonApiId;
+import io.katharsis.resource.annotations.JsonApiResource;
+
+/**
+ * Created by nickmitchell on 1/6/17.
+ */
+@JsonApiResource(type = "lang-locales")
+public class Locale {
```

Comment:
Sorry, but I need to make my point a bit more on `@Lombok` -- this whole class could be reduced to 5 lines for identical functionality